src/avatar/styled-components.js

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import {styled} from '../styles';
+import type {StylePropsT} from './types';
+
+function getSize(props: StylePropsT) {
+  const {$size, $theme} = props;
+
+  const defaultSize = $theme.sizing.scale1000;
+  const size = $size || defaultSize;
+  return $theme.sizing[size] || defaultSize;
+}
+
+export const Avatar = styled('img', (props: StylePropsT) => {
+  const themedSize = getSize(props);
+
+  return {
+    borderRadius: '50%',
```

Comment:
we may need to add `box-sizing: border-box;` in case user adds overrides of border and padding, so 50% will still work

Reply:
updated